content/applications/marketing/events/overview/create.rst

```diff
@@ -0,0 +1,298 @@
+well.
+
+Also, if any specific documents are required (or helpful) for this event invite, they can be
+sent along with this email, by clicking :guilabel:`Attachments`, and adding the appropriate
+document(s).
+
+If *SMS* is selected as the :guilabel:`Mailing Type`, a :guilabel:`Responsible` can be designated,
+and the option to :guilabel:`Include an opt-out link` is available.
+
+Send event invite to recipients
+-------------------------------
+
+If the selected :guilabel:`Mailing Type` is *Email*, there are three options to send the invite:
+:guilabel:`Send`, :guilabel:`Schedule`, and :guilabel:`Test`.
+
+The :guilabel:`Send` option sends the invite right away. The :guilabel:`Schedule` option reveals a
+pop-up, in which a scheduled date/time can be selected for the email to be sent. The
+:guilabel:`Test` option reveals a :guilabel:`Test Mailing` pop-up, in which specific recipient
+email addresses are entered for Odoo to send them the current version of the mailing, which tests
+the effectiveness of the event invite.
+
+.. image:: create/send-schedule-test.png
+   :align: center
+   :alt: View of the send, schedule, and test options when sending an event invite via email.
+
+If the selected :guilabel:`Mailing Type` is *SMS*, there are four options to send the invite:
+:guilabel:`Put in Queue`, :guilabel:`Send Now`, :guilabel:`Schedule`, and :guilabel:`Test`.
+
+The :guilabel:`Put in Queue` option schedules an SMS message to all recipients (that match the
+designated rules, if any) in the near future. Clicking :guilabel:`Put in Queue` reveals a pop-up,
+requiring confirmation. When confirmed, a blue banner appears on the event invite template form,
+indicating that the SMS will be sent later that day.
+
+.. note::
+   The :guilabel:`Send Now`, :guilabel:`Schedule`, and :guilabel:`Test` options all function the
+   same as the *Email* :guilabel:`Mailing Type`.
+
+.. image:: create/queue-send-schedule-test.png
+   :align: center
+   :alt: View of the put in queue, send now, schedule, and test options when sending an event invite
+         via SMS.
+
+Publish events
+==============
+
+If events aren't published, attendees won't be able to access it. To publish an event page,
+navigate to the event page on the website, from the back end or the front end, and publish it.
+
+If starting from the back end, go to the event form, and click the :guilabel:`Go to Website` smart
+button to reach the event page on the website (on the front end). If starting from the front end,
+simply navigate to the event page that needs to be published.
+
+No matter the route, an event page can only be published from the front end (on the website, as an
+admin). In the upper right corner of the event page on the website, toggle the switch from
+the red :guilabel:`Unpublished` status to the green :guilabel:`Published` status. Doing so
+instantly makes the event page accessible on the website.
```

Comment:
Read a couple times and was confusing to me.
Suggested rewrite:

```suggestion
Until an event is published, it will remain hidden from public view on the website and registering
for it will not be possible. To publish an event, navigate to it either from the back end of Odoo
through the :guilabel:`Events` application, or access the hidden event page through the front end as
either a priveliged user or administrator.

If navigating from the back end, go to the event form, and click the :guilabel:`Go to Website` smart
button to reach the event page on the website (on the front end). If starting from the front end,
simply navigate to the event page that needs to be published.

No matter the route, an event page can only be published from the front end. In the upper right
corner of the event page on the website, toggle the switch from the red :guilabel:`Unpublished`
status to the green :guilabel:`Published` status. Doing so instantly makes the event page accessible
to the public on the website.
```
```